proto/eth/v1alpha1/slasher.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package ethereum.eth.v1alpha1;
+
+import "google/protobuf/empty.proto";
+import "proto/eth/v1alpha1/beacon_block.proto";
+import "proto/eth/v1alpha1/attestation.proto";
+
+option go_package = "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1;eth";
+
+// Slasher service API
+// 
+// Slasher service provides an interface for validators and beacon server to query
+// to subscribe for slashable events on the network as well as to make sure that the 
+// attestation or proposal they are going to submit to the network are not going to 
+// produce a slashable event.
+service Slasher {
+
+    
```

Comment:
Eliminate the white space here and line 18

Reply:
No need to regen protos after that

Reply:
removed